core/src/main/scala/org/apache/spark/TestUtils.scala

```diff
@@ -192,6 +192,20 @@ private[spark] object TestUtils {
     assert(listener.numSpilledStages == 0, s"expected $identifier to not spill, but did")
   }
 
+  /**
+   * Asserts if exception message doesn't contain the message. Please note this checks all
```

Comment:
"that exception message contains"

At least that's what the code seems to be doing.

Reply:
I was hesitating what you mean here but now I see the point.
`Asserts if` == `throws exception if` was my intention but reading this back as an english sentence is just bad. Fixed.